include/mbedtls/ssl.h

```diff
@@ -3309,8 +3309,16 @@ int mbedtls_ssl_session_load(mbedtls_ssl_session *session,
  *                 to determine the necessary size by calling this function
  *                 with \p buf set to \c NULL and \p buf_len to \c 0.
  *
+ * \note           For TLS 1.3 sessions, this feature is supported only if the
+ *                 MBEDTLS_SSL_SESSION_TICKETS configuration option is enabled,
```

Comment:
Should this be `MBEDTLS_SSL_SESSION_TICKETS` to create a Doxygen link? (same on line 3320)